CMakeLists.txt

```diff
@@ -191,20 +191,18 @@ endif()
 # Use Wayland for window creation
 #--------------------------------------------------------------------
 if (_GLFW_WAYLAND)
-    find_package(ECM REQUIRED NO_MODULE)
-    list(APPEND CMAKE_MODULE_PATH "${ECM_MODULE_PATH}")
 
-    find_package(Wayland REQUIRED Client Cursor Egl)
-    find_package(WaylandScanner REQUIRED)
-    find_package(WaylandProtocols 1.15 REQUIRED)
+    include(FindPkgConfig)
+    pkg_check_modules(Wayland REQUIRED
+        wayland-client>=0.2.7
+        wayland-cursor>=0.2.7
+        wayland-egl>=0.2.7
+        xkbcommon)
```

Comment:
Is there a reason to change from CMake packages to pkg-config? CMake packages also supports specifying the minimum version.

Reply:
It works by default for a host build, easy to cross compile on host (set PKG_CONFIG_DIR env variable), and it doesn't require an extra dependency.